Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -31,17 +31,12 @@ public override bool CanAccessViaStorage(EntityUid user, EntityUid target)
             if (!_container.TryGetContainingContainer(target, out var container))
                 return false;
 
-            if (!TryComp(container.Owner, out StorageComponent? storage))
+            var ev = new CanInteractWhileInsideContainerEvent(user, container);
+            RaiseLocalEvent(target, ev);
+            if (!ev.Handled)
                 return false;
 
-            if (storage.Container?.ID != container.ID)
-                return false;
-
-            if (!TryComp(user, out ActorComponent? actor))
-                return false;
-
-            // we don't check if the user can access the storage entity itself. This should be handed by the UI system.
-            return _uiSystem.SessionHasOpenUi(container.Owner, StorageComponent.StorageUiKey.Key, actor.PlayerSession);
```

Comment:
Its a quirk with how the sticky system works. It places the c4 (or whatever) inside a container on the wall (or whatever its stuck to) so by default it can't be interacted with since you can't access the container.

Sloth made a big update to the interaction system already though so I need to figure out how it works now to fix conflicts.

Reply:
Decided to just wait on the interaction changes for another PR since this *will* work without it, it just doesn't feel quite as clean.